api/api.go

```diff
@@ -84,6 +96,11 @@ func (api Api) deploy(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
+	if err := validateDeploymentRequirements(api.FasitUrl, deploymentRequest); err != nil {
+		glog.Errorf("Validating requirements for deployment failed: %s", err)
```

Comment:
return here?

Reply:
That would mean that every application deployed through naisd must be tied to an FASIT environment and must have been defined in FASIT.

Do we really want this hard dependency on FASIT?

At the very least it makes testing NAISD in minikube for instance a bit cumbersome.

Reply:
I agree. No return :)